src/pyasm/web/monitor.py

```diff
@@ -495,6 +495,8 @@ def run(self):
         
 
         # NOTE: not sure why we have to do a batch here
+        from pyasm.security import Site
```

Comment:
Hey! Can you remove this from your pull request? We are only keeping this internally.